Trinap/Sources/Domain/UseCases/Contact/FetchContactUseCase.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  FetchContactUseCase.swift
+//  Trinap
+//
+//  Created by Doyun Park on 2022/12/06.
+//  Copyright © 2022 Trinap. All rights reserved.
+//
+
+import Foundation
+
+import RxSwift
+
+protocol FetchContactUseCase {
+    func fetchContacts() -> Observable<[Contact]>
+    func fetchDetailContact(contactId: String) -> Observable<Contact>
+}
+
+final class DefaultFetchContactUseCase: FetchContactUseCase {
+    
+    private let userRepository: UserRepository
+    var disposeBag = DisposeBag()
+    init(userRepository: UserRepository) {
+        self.userRepository = userRepository
+    }
+    
+    func fetchContacts() -> Observable<[Contact]> {
+        return userRepository.fetchContact()
+    }
+    
+    func fetchDetailContact(contactId: String) -> Observable<Contact> {
+        return userRepository.fetchDetailContact(contactId: contactId)
+    }
+}
```

Comment:
컨벤션 맞춰주시면 감사하겠습니다..!

```suggestion
final class DefaultFetchContactUseCase: FetchContactUseCase {
    
    // MARK: - Properties
    private let userRepository: UserRepository
    var disposeBag = DisposeBag()
    
    // MARK: - Initializer
    init(userRepository: UserRepository) {
        self.userRepository = userRepository
    }
    
    // MARK: - Methods
    func fetchContacts() -> Observable<[Contact]> {
        return userRepository.fetchContact()
    }
    
    func fetchDetailContact(contactId: String) -> Observable<Contact> {
        return userRepository.fetchDetailContact(contactId: contactId)
    }
}
```